lib/embulk/input/mixpanel.rb

```diff
@@ -162,10 +162,10 @@ def run
         @dates.each_slice(task[:slice_range]) do |dates|
           ignored_record_count = 0
           unless preview?
-            Embulk.logger.info "Fetching data from #{dates.first} to #{dates.last} ..."
+            Embulk.logger.info "Fetching data from #{@dates.first} to #{dates.last} ..."
```

Comment:
Use both of `@dates` and `dates` in same block is confusing?

Reply:
Yes, I will change it to a different name